frontend/src/scenes/surveys/SurveyCustomization.tsx

```diff
@@ -50,14 +53,26 @@ export function Customization({
                         value={appearance?.backgroundColor}
                         onChange={(backgroundColor) => onAppearanceChange({ ...appearance, backgroundColor })}
                         disabled={!surveysStylingAvailable}
+                        className={clsx(
+                            surveyErrors?.appearance?.backgroundColor ? 'border-danger' : 'ignore-error-border'
```

Comment:
those `ignore-error-border` are necessary because `appearance` is a JSON field, so validation was being applied to ALL the fields when there was an error.

instead I'm handling each error individually, and added the `ignore-error-border` to other ones to ignore it, as there are no other validations in appearances